test/object-store/sync/client_reset.cpp

```diff
@@ -237,8 +237,10 @@ TEST_CASE("sync: client reset", "[client reset]") {
         ++after_callback_invocations;
         REQUIRE(before);
         REQUIRE(before->is_frozen());
-        REQUIRE(before->read_group().get_table("class_object"));
-        REQUIRE(before->config().path == local_config.path);
+        if (!before->config().is_schema_additive()) {
```

Comment:
This is totally unclear, I don't get why this is failing. There is no obvious, reason for it.